code/modules/events/radiation_storm.dm

```diff
@@ -60,6 +60,9 @@
 				else
 					randmutg(H) // Applies good mutation
 					domutcheck(H,null,MUTCHK_FORCED)
+		//Just having it apply to cats now
+		if(istype(C,/mob/living/simple_animal/cat))
+			C.apply_effect((rand(30,60)),IRRADIATE,blocked=0)
```

Comment:
This is bad OOP. Considering what happens above, we have to think in an "expansion"-y sense, in that if we want to add more mobs with radiation mechanics later on, we should make it easy to do so. This is true for pretty much all mechanics you add. Here's what I suggest:

You see how we keep adding istype checks here? That's bad, we can simplify it a lot by just making a proc on mob/living that's called like `apply_radiation_effects`. Then we can change it for all mobs and it's much cleaner than this.